scripts/azure-pipelines/build.ps1

```diff
@@ -0,0 +1,48 @@
+﻿# Builds Mumble using the given environment and build script.
+# The configuration we build with is adjusted to be close to
+# our release builds.
+#
+# Configuration variables used from environment:
+#
+#  AGENT_BUILDDIRECTORY       - The local path on the agent where all folders
+#                               for a given build pipeline are created
+#                               (e.g. "D:\a\1")
+#  BUILD_SOURCESDIRECTORY     - The local path on the agent where the
+#                               repository is downloaded.
+#                               (e.g. "D:\a\1\s")
+#  MUMBLE_ENVIRONMENT_DIR     - The local path where the build environment
+#                               will be stored (e.g. "C:\MumbleBuild").
+#  MUMBLE_ENVIRONMENT_VERSION - Full build environment version
+#                               (e.g. win64-static-no-ltcg-1.3.x-2017-02-02-ec94ddb-790).
+#                               Must match .7z and extracted folder name.
+#  MUMBLE_BUILDSCRIPT         - Path to required build script cmd file. Relative to build
+#                               environment's "mumble-releng\buildscripts" path.
+#                               (e.g. "1.3.x/buildenv-win64-static.cmd")
+#
+
+$MUMBLE_BUILD_DIR = $env:AGENT_BUILDDIRECTORY
+$MUMBLE_SOURCE_DIR = $env:BUILD_SOURCESDIRECTORY
+$MUMBLE_BUILDENV_DIR = Join-Path $env:MUMBLE_ENVIRONMENT_DIR $env:MUMBLE_ENVIRONMENT_VERSION
+$MUMBLE_BUILDSCRIPT = Join-Path $MUMBLE_BUILDENV_DIR "mumble-releng\buildscripts\$env:MUMBLE_BUILDSCRIPT"
+
+$env:MUMBLE_EXTRA_QMAKE_CONFIG_FLAGS = ""
+
+# We do not sign the Azure Pipelines CI builds, so we must disable
+# uiaccess elevation. Also no intermediary signing is wanted.
```

Comment:
Why?

Reply:
Why we don't sign it or why we can't elevate if not signed?